doc/src/site/sphinx/tutorials/tutorials.rst

```diff
@@ -73,6 +73,13 @@ How to...
    sw_azure_dbc
    pysparkling_azure_dbc
 
+**Google Cloud Dataproc**
+
+.. toctree::
+   :maxdepth: 2
+
+   sw_google_cloud_dataproc
```

Comment:
I would put this page into the Deployment section

Reply:
Hmm… If I move this to Deployment, then should the following also be moved to Deployment?
- Using SW with Amazon EMR
- Using SW with Microsoft Azure HDI
- Running Sparkling on Databricks Azure
- Running RSparkling on Databricks Azure
- Running PySparkling on Databricks Azure

Reply:
Yes, exactly! I would always name the section with the type of platform where we are deploying. 

Thanks!

Reply:
Hmm... I don't know how to get Sphinx to recognize captions for lower-level headings. Meaning, if the section (main heading) is called Deployment, then I'm not able to make subsection captions.